README.md

```diff
@@ -1355,6 +1356,11 @@ Pre-fills the identifier field with the previously used username.
 Defaults to `true`.
 Automatically focuses the first input field of any form when displayed. 
 
+#### features.disableAutocomplete
+
+Defaults to `false`.
+Sets the autocomplete attribute on Input fields to `off`.
```

Comment:
cc: @brentschaus-okta 

Can you review this and recommend any changes (if necessary).

Reply:
Looks good.